resolve_stream.py

```diff
@@ -157,8 +322,18 @@ def main():
     refdes = options['<refdes>']
     method = options['<method>']
     stream = options['<stream>']
-
-    if not all((refdes, method, stream)):
+    param = options['<parameter>']
+
+    if all((refdes, method, stream, param)):
+        qp = lookup_parameter(param, stream, refdes, method)
+        required, unresolved = fully_resolve_parameter(qp)
```

Comment:
In the pathological case when unresolved is not empty, do we want to print something to indicate the error?